import-express.html

```diff
@@ -40,11 +40,18 @@ <h2>Import - Express</h2>
                 <div class="section-cols">
                     <div class="section-col">
                         <form class="import-form">
-                            <sp-field-label for="import-url" required>URL</sp-field-label>
-                            <sp-textfield class="option-field" id="import-url" type="url">
-                                <sp-help-text slot="help-text">https://www.example.com</sp-help-text>
-                                <sp-help-text slot="negative-help-text">Please enter a valid URL.</sp-help-text>
-                            </sp-textfield>
+                            <div id="express-url-inputs">
+                                <div>
+                                    <sp-field-label for="import-url" required>URL</sp-field-label>
+                                    <sp-textfield class="option-field" id="import-url" type="url">
+                                        <sp-help-text slot="help-text">https://www.example.com</sp-help-text>
+                                        <sp-help-text slot="negative-help-text">Please enter a valid URL.</sp-help-text>
+                                    </sp-textfield>
+                                </div>
+                                <sp-button id="express-open-url-button" title="Open URL in new tab.">
+                                    <sp-icon-open-in>Open</sp-icon-open-in>
```

Comment:
Since this button opens an external URL in a new tab the `link out` icon should be used. Also, the screenshot doesn't show the icon, is the screenshot just older?

Reply:
Without updating js/dist, the icon doesn't show up.  Once a `npm run build` is done, it shows up.

Reply:
Once this PR is merged https://github.com/arumsey/helix-importer-ui/pull/10 can be updated in order to generate a new set of dependent modules.

Reply:
Done.